### PR TITLE
builtin: Support User() and Current_User()

### PR DESCRIPTION
**Pull request by shenli**

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -1888,6 +1889,17 @@ FunctionCallConflict:
 			return 1
 		}
 	}
+|	"CURRENT_USER"
```
Comment (siddontang) [inline parser/parser.y]: no parser rule for `current_user()` ?

Comment (shenli) [inline parser/parser.y]: Current_user() is parsed in "FunctionNameConflict '(' ExpressionListOpt ')'"

**Comment by siddontang** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**